### PR TITLE
feat(styling): add styled components

### PR DESCRIPTION
**Pull request by alejandronanez**
Adding `styled-components` initial work! 🙌

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -20,18 +14,17 @@ import { determineLanguage } from 'locale';
...
-    alignItems: 'center',
-  },
-  logo: {
-    width: 100,
-    height: 100,
-  },
-});
+const Container = styled.View`
+  align-items: center;
+  background-color: ${() => colors.white}
```
Comment (chinesedfan) [inline App.js]: @alejandronanez I am not sure. Can it just be `background-color: 'white';` ?

Comment (andrewda) [reply App.js]: Doing it this way makes it so we can change the white color everywhere by just changing the a single value. However, why can't we just do `${colors.white}` @alejandronanez?

Comment (alejandronanez) [reply App.js]: Hey @chinesedfan 
- We should avoid at all costs using plain strings for colors, that's not a good practice.
- We already have a `colors` file that we should keep using.

Comment (chinesedfan) [reply App.js]: @alejandronanez Yes, my point is the anonymous function, as @andrewda mentioned. (You merged so quickly. 😆 )

Comment (alejandronanez) [reply App.js]: @andrewda you're right! There you go https://github.com/gitpoint/git-point/pull/497/files

Comment (alejandronanez) [reply App.js]: Oh, I put a fix for that too @chinesedfan! :)

Comment (ericadamski) [reply App.js]: Understanding that this is already merged, there is no need for a function here, you can just use string interpolation

```javascript
background-color: ${colors.white};
```

**Pull request merged**